README.md

```diff
@@ -121,16 +113,10 @@ for more information about how to generate these.
 
 ```puppet
 node 'broker1.example.com' {
-  class { '::mcollective':
-    middleware         => true,
-    middleware_hosts   => [ 'broker1.example.com' ],
-    middleware_ssl     => true,
-    securityprovider   => 'ssl',
-    ssl_client_certs   => 'puppet:///modules/site_mcollective/client_certs',
-    ssl_ca_cert        => 'puppet:///modules/site_mcollective/certs/ca.pem',
-    ssl_server_public  => 'puppet:///modules/site_mcollective/certs/server.pem',
-    ssl_server_private => 'puppet:///modules/site_mcollective/private_keys/server.pem',
-  }
+  # Please see
+  # https://github.com/puppetlabs/puppetlabs-mcollective/blob/master/examples/mco_profile/manifests/middleware/activemq.pp
+  # for this as the changes to template and neccessity to build a
+  # keystore makes this quite complex.
```

Comment:
The intent of the change is just to update the docs to reflect PR #157's gutting of the module.  This was a working complete example before, but it can't be made a working example without inlining all this code or equivalents into the `node 'broker1.example.com'` stanza:  https://github.com/puppet-community/puppet-mcollective/blob/master/examples/mco_profile/manifests/middleware/activemq.pp#L63-L101

Can update the PR to do that, but it'll make this sample very verbose.